src/libs/actions/IOU.js

```diff
@@ -126,6 +127,9 @@ function buildOnyxDataForMoneyRequest(
     optimisticPolicyRecentlyUsedTags,
     isNewChatReport,
     isNewIOUReport,
+    policy,
+    policyTags,
+    policyCategories,
```

Comment:
Not your fault, but these argument lists give me nightmares!

Reply:
Same.